local-proxy/README.md

```diff
@@ -0,0 +1,39 @@
+Flipt Local Proxy
+------------
+
+This lab section's intent is to explore the ways that clients can achieve fast evaluations from Flipt for their feature flags. The use case here is that users would like data locality for their feature flag state, so they do not have to worry about network + application time to get a feature flag evaluation value especially for their most critical application paths.
+
+### S3 Replication
+
+This project is housed under the `s3-replication` directory. The purpose is for a user to run their application which depends on feature flags with a sidecar Flipt process that pulls data from an S3 bucket. The evaluation data here can then be accessed over `localhost` talking to the Flipt sidecar.
+
+Lets get started!
+
+Prerequisites:
+- [docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [minikube](https://minikube.sigs.k8s.io/docs/)
+
+This project uses kubernetes to deploy the following:
+- `minio`: Object store that has an S3 compatible API
+- `flipt-master`: Serves as the main Flipt application, this is where users will be accessing the UI to make relevant changes
+- `sample-app`: Serves as the pod with Flipt running as a sidecar, `flipt-sidecar`, pulls its data from S3 (main application can be one of your containers within the pod)
+- `flipt-exporter`: CronJob that runs on a 1 minute interval that exports data out of the Flipt master, and puts those changes onto the object store
+
+To access the `flipt-master` API, you can use kubernetes to port-forward the service:
+
+```bash
+$ kubectl port-forward svc/flipt-master --namespace default 8080:8080
+```
+
+This is so you can use the API to access the UI, and add data to Flipt as necessary.
+
+The same goes for the sidecar:
+
+```bash
+$ kubectl port-forward svc/sample-app --namespace default 8080:8080
+```
+
+> The difference for the sidecar is that it will be in `readonly` mode, due to it sourcing its data from a file system.
+
+There exists a deploy script `deploy.sh` that will provision the cluster for you and start all the necessary deployments and services for the whole application described above.
```

Comment:
mind renaming this script to `scripts/start` to match our existing pattern in the other labs?